Modules/Tests/YosemiteTests/Storage/PersistedProductConversionsTests.swift

```diff
@@ -0,0 +1,203 @@
+import Foundation
+import Testing
+@testable import Yosemite
+
+struct PersistedProductConversionsTests {
+
+    @Test("PersistedProduct init(from:) maps all POSProduct fields")
+    func test_product_init_from_posProduct_maps_all_fields() throws {
```

Comment:
super nit: with Swift Testing, we probably don't need the `test_` prefix in the function name as each test case is already marked with `@Test`.

Reply:
I'll update them. TBH I think we could also leave behind the snake case, if we use descriptive names as arguments to `@Test` – they display much nicer anyway, at least within Xcode. I'm not sure whether CI pays any attention to them though.